libglusterfs/src/glusterfs/mem-types.h

```diff
@@ -120,6 +120,8 @@ enum gf_common_mem_types_ {
     gf_common_mt_mgmt_v3_lock_timer_t, /* used only in one location */
     gf_common_mt_server_cmdline_t,     /* used only in one location */
     gf_common_mt_latency_t,
-    gf_common_mt_end
+    gf_common_mt_pmap_reg_t,  /* used only in one location */
```

Comment:
nit: I wonder why we add more memory types to the common list, while we only use it in one place. The idea is that those that are used in a single location will actually be defined there, not across all xlators.